VideoLocker/src/main/java/org/edx/mobile/view/MyVideosActivity.java

```diff
@@ -2,19 +2,43 @@
 
 import android.os.Bundle;
 import android.support.design.widget.TabLayout;
-import android.support.v4.view.PagerAdapter;
 import android.support.v4.view.ViewPager;
 import android.view.View;
 import android.view.WindowManager;
 
 import org.edx.mobile.R;
 import org.edx.mobile.base.BaseVideosDownloadStateActivity;
+import org.edx.mobile.logger.Logger;
 import org.edx.mobile.module.analytics.ISegment;
 import org.edx.mobile.view.adapters.StaticFragmentPagerAdapter;
 
-public class MyVideosTabActivity extends BaseVideosDownloadStateActivity {
+public class MyVideosActivity extends BaseVideosDownloadStateActivity implements MyRecentVideosFragment.MyVideosTabListener {
 
     private View offlineBar;
+    private final Logger logger = new Logger(MyVideosActivity.class.getName());
+    private StaticFragmentPagerAdapter adapter;
+
+    public void refreshMyAllVideosTab() {
+        // TODO Pick one of these
```

Comment:
@BenjiLee: You have modified or replaced the commit in which you commented, and subsequently I can't actually see which approached you were thinking about. However, how about using the event bus? It would probably make things a lot simpler and modular.